client/interfaces.go

```diff
@@ -313,6 +313,7 @@ type InstanceServer interface {
 	GetClusterMember(name string) (member *api.ClusterMember, ETag string, err error)
 	UpdateClusterMember(name string, member api.ClusterMemberPut, ETag string) (err error)
 	RenameClusterMember(name string, member api.ClusterMemberPost) (err error)
+	AddClusterMember(name string) (op Operation, err error)
```

Comment:
Should be `CreateClusterMember` and take a `api.ClusterMembersPost` struct as input to line up with the other functions.